src/backend/vulkan/src/physical_device.rs

```diff
@@ -582,6 +582,8 @@ impl PhysicalDeviceInfo {
             requested_extensions.push(vk::KhrGetDisplayProperties2Fn::name()); // TODO NOT NEEDED, RIGHT?
         }
 
+        requested_extensions.push(vk::ExtDisplayControlFn::name());
```

Comment:
During the `DeviceExtensionFunctions`, if the requested extensions is not available, the `DeviceExtensionFunctions.display_control` will be None. The functions that take advantage of the extension try to get the it from the device and if it is None, they will return `UnsupportedFeature` error. At least this is the expected behaviour, do i miss something?

Reply:
So you are relying on this code:
```rust
            if !unsupported_extensions.is_empty() {
                warn!("Missing extensions: {:?}", unsupported_extensions);
            }
```
This code is not something to rely on. It will warn, and seeing a warning on a system that doesn't support the display control would be confusing for people who don't need it.

Reply:
Right, ok. A solution could be something like:
```
        if self.supports_extension(vk::ExtDisplayControlFn::name()){
            requested_extensions.push(vk::ExtDisplayControlFn::name());
        }
```
In this way if the extension is not supported, it is not requested. So, if the extension is not enabled, `DeviceExtensionFunctions.display_control` will be `None` and it should have the same behaviour as before.
What do you think?

Reply:
Sounds good. Ultimately what we see here is the lack of support for instance extensions in gfx-hal, but that should be left for another time to solve.